test/test_kubeclient.rb

```diff
@@ -403,6 +515,27 @@ def test_api_bearer_token_failure
     assert_equal(response, exception.response)
   end
 
+  def test_api_bearer_token_failure_raw
+    error_message =
+      '"/api/v1" is forbidden because ' \
+      'system:anonymous cannot list on pods in'
+    response = OpenStruct.new(code: 401, message: error_message)
```

Comment:
[unrelated to this PR]
looks like `code:` here is either unnecessary or should be 403.
I'm not quite sure why this test asks stub `.to_raise(Kubeclient::HttpError...)` instead of `.to_return(..., status: 401)`.
Similar in `test_api_bearer_token_failure` from which this was copied, and several other tests.